egs/librimix/ConvTasNet/local/conf.yml

```diff
@@ -0,0 +1,37 @@
+encoder:
+  enc_name : free
+decoder:
+  dec_name : free
+
```

Comment:
How about sticking to ConvTasnet for a baseline recipe? So we assume encoder/decoder use free filterbanks.

Reply:
Sure